uui/components/datePickers/__tests__/DatePicker.test.tsx

```diff
@@ -103,4 +109,42 @@ describe('DatePicker', () => {
         expect(mocks.onValueChange).toHaveBeenCalledWith('2017-01-31');
         expect(dom.input.value).toEqual('31-01-2017');
     });
+
+    it('should support entering date from keyboard in default format', async () => {
+        const { dom, mocks } = await setupDatePicker({ value: null, format: DATE_FORMAT_DEFAULT });
+        expect(dom.input.value).toEqual('');
+        fireEvent.change(dom.input, { target: { value: '2017-01-22' } });
```

Comment:
You are entering date not in default format.

Reply:
Yes, I intended to check that after outside click format will be changed to one we provided.